environment.yml

```diff
@@ -10,6 +10,7 @@ dependencies:
   - pytest-xdist
   - pytest-cov
   - coverage
+  - openff-toolkit
```

Comment:
add `jaimergp/label/unsupported-cudatoolkit-shim` to channels above and `channel-priority: strict` to the conda-incubator/setup-miniconda@v2 in the ci.yaml file?

Reply:
Nah I think we're just sucking it up and installing cuda

Reply:
are we treating environment.yml as something users will use?

Reply:
I would think perhaps code contributors, but not users -- env for users is defined by the recipe in the feedstock

Reply:
In that case I'm not sure why we would needlessly slowdown CI by not including the shim?